### ISSUE TITLE
"Install configured modules with drush" hangs when misconfigured

### ISSUE THREAD

**Issue by ccjjmartin** [geerlingguy/ansible-role-drupal#48]
I misconfigured my config.yml but found out that if you have this type of configuration:

drupal_composer_dependencies: []

drupal_enable_modules: [ 'devel' ]

That this task will hang forever (I waited over an hour first time around).  Made it slightly hard to debug.  So this would probably be a feature request to throw an error instead of hanging, not sure how hard this would be to implement.

**Comment by geerlingguy** [comment]
Strange... it should still work in the case you show above, without delay, but maybe there's a conditional check that's wrong?

Which task is hanging, specifically, and what changed did you have to make to fix it?

**Comment by ccjjmartin** [comment]
@geerlingguy Thank you for responding so many years ago.  I missed the ping and now I won't be able to reproduce this anymore.  So closing it was the right thing to do here.

I took a look at the current code anyway and I agree with you that this should work in theory.  When I opened this if I am remembering correctly I believed that the issue might be the empty array in the `drupal_composer_dependencies` line because I think the fix was to add `devel` to the array but the current code shows that the `drupal_composer_dependencies` variable actually defaults to an empty array so this is likely a non issue.

Reason why I believe it is a non issue:
https://github.com/geerlingguy/ansible-role-drupal/blob/master/tasks/build-composer.yml#L36

Thanks again for responding to this.
